<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmpleadoCajaWindow</class>
 <widget class="QMainWindow" name="EmpleadoCajaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel de Caja</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="topbar">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="status_label1">
         <property name="text">
          <string>BD Remota 1: Desconectada</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="status_label2">
         <property name="text">
          <string>BD Remota 2: Desconectada</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="logout_button">
         <property name="text">
          <string>Cerrar Sesión</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pagos_tab">
       <attribute name="title">
        <string>Pagos</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="reservas_list"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Monto Recibido:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="monto_edit"/>
          </item>
          <item>
           <widget class="QPushButton" name="aprobar_pago_button">
            <property name="text">
             <string>Aprobar Pago</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="caja_tab">
       <attribute name="title">
        <string>Caja del Día</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="total_caja_label">
          <property name="text">
           <string>Total efectivo del día: $0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="transacciones_count_label">
          <property name="text">
           <string>Transacciones: 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="promedio_caja_label">
          <property name="text">
           <string>Promedio: $0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="transacciones_list"/>
        </item>
        <item>
         <widget class="QPushButton" name="cerrar_caja_button">
          <property name="text">
           <string>Cerrar Caja</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
